frontend/src/components/NavBarSuggest/NavBarSuggest.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState } from "react";
+import "./style.scss";
+
+export default function NavBarSuggest() {
+  const [isActive, setIsActive] = useState("idea");
+
+  return (
+    <div className="navBarS">
+      <button
+        type="button"
+        className="firstB"
+        style={{
+          backgroundColor: isActive === "idea" ? "#70af90" : "#70af8f60",
+          boxShadow: isActive === "idea" ? "0 20px 60px 0 black" : "unset",
+          cursor: "pointer",
+        }}
```

Comment:
The style tags are to be avoided. Is it possible to rework this?

Reply:
removed

Reply:
removed